bundown/src/cli/index.ts

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bun
+
+import { usage } from './usage'
+import { log } from './log'
+import { sdk } from '../sdk/index'
+import { sync, run } from '../core/index'
+
+import { $, semver } from 'bun'
+import { parseArgs } from 'node:util'
+import { match, P } from 'ts-pattern'
+
+export async function cli (params: { args: string[] }): Promise<void> {
+  const args = parseArgs({
+    args: params.args,
+    strict: true,
+    allowPositionals: true,
+    options: {
+      // v1
+      help:    { short: 'h', type: 'boolean', multiple: false },
+      version: { short: 'v', type: 'boolean', multiple: false },
+      print:   { short: 'p', type: 'boolean', multiple: false },
+      tag:     { short: 't', type: 'string',  multiple: true  },
+      // v2
+      file:    { short: 'f', type: 'string',  multiple: false },
+      runtime: { short: 'r', type: 'string',  multiple: false },
+      os:      {             type: 'string',  multiple: false },
+    }
+  })
+  const expectedBunVersion = '^1.0.27'
```

Comment:
Can we please move this back to the `package.json` import and set it back to `1.0.24`?

The intent here is to require the minimum viable Bun version that can run Bundown, which so far means the earliest version supporting the `$` Bun Shell. That information should just come from the `package.json` dependency.

This is also why the `$` import was happening after the throw.

Reply:
Sounds good.